docs/configuring-entities.md

```diff
@@ -0,0 +1,197 @@
+
+```yaml
+- name: Send dispatch for service-catalog
+  uses: operate-first/service-catalog/.github/actions/docs-dispatch@main
+  with:
+    # Root of the repository
+    docs_path: '.'
+    # Name of your entity in the catalog
+    entity_name: 'service-catalog'
+    # Type of your entity in the catalog
+    entity_kind: 'Component'
+    # This can always be ${{ github.repository }}
+    repository: ${{ github.repository }}
+    # GitHub token that requires metadata:read and contents:read&write permissions to service-catalog repository
+    token: ${{ secrets.SESHETA_TOKEN }}
+```
+
+If the documentation is not located in the root of the repository, a filter definition is required using [glob expresions][15]:
+
+```yaml
+- name: Send dispatch for service2
+  uses: operate-first/service-catalog/.github/actions/docs-dispatch@main
+  with:
+    # Glob expression, filtering where your documentation root is
+    filter: 'service2/**'
+    # Directory path to your documentation folder relative to the repository root
+    docs_path: 'service2'
+    entity_name: 'service2'
+    entity_kind: 'Component'
+    repository: ${{ github.repository }}
+    token: ${{ secrets.SESHETA_TOKEN }}
+```
+
+<!--Mermaid diagram showing techdocs architecture somewhere here -->
+[Documentation reference][13]
+
+## GitHub Insights/Security Insights
+
+For the insights plugins to work it is required to put `github.com/project-slug` annotation to your components definition. The value of this annotation is a GitHub project slug which consists of the organization/user and the repository name, see the example below.
+
+```yaml
+annotations:
+    github.com/project-slug: operate-first/service-catalog
+```
+
+As long as the repository is public `GitHub Insights` will be shown, for one exception. If the repository is private and the GitHub account that is used for the login and has the permission for the repository, insights will be also shown. This applies to `Security Insights` too, if the GitHub account has the correct permissions to view the repository security insights, the plugin will show them.
+
+[Documentation reference for GitHub Insights][3]
+
+[Documentation reference for GitHub Security Insights][4]
+
+## ADR plugin
+
+ADR plugin requires an annotation to be present in your component definition. Right now the only value that is accepted by the annotation is a GitHub repository link to the folder which contains your ADRs, see the example below.
+
+```yaml
+annotations:
+    backstage.io/adr-location: https://github.com/operate-first/blueprint/tree/main/adr
+```
+
+[Documentation reference][5]
+
+## Grafana plugin
+
+The grafana plugin can display either links to dashboards or alerts on the overview page for a configured tag in the components annotations. For example:
```

Comment:
This is not "either .. or" case. The plugin is not exclusive - It's A OR B, not A XOR B.

```suggestion
Grafana plugin can display links to Grafana dashboards and alerts on the overview page. It is configured via an annotation on entity declaration. This annotation describes which Grafana tag should be queried for dashboards and AlertManager tag for alerts.
```

Reply:
Resolved in the latest commit.